source/_data/themes.yml

```diff
@@ -3320,6 +3320,20 @@
     - widgets
     - responsive
     - dark/light
+- name: Web-Nary
+  description: A Fully Customised Hexo Flat Modular Theme for Bloging and More ...
+  link: https://github.com/BRAVO68WEB/hexo-webnary-theme/
```

Comment:
Would you mind renaming the theme to `hexo-theme-webnary`? Thanks.

Reply:
Ok Sure